lib/ziti-tunnel/tunnel_udp.c

```diff
@@ -245,7 +245,7 @@ u8_t recv_udp(void *tnlr_ctx_arg, struct raw_pcb *pcb, struct pbuf *p, const ip_
     io->close_fn = intercept_ctx->close_fn ? intercept_ctx->close_fn : tnlr_ctx->opts.ziti_close;
     io->tnlr_io->idle_timeout = UDP_TIMEOUT;
 
-    TNL_LOG(INFO, "intercepted address[%s] client[%s] service[%s]", io->tnlr_io->intercepted, io->tnlr_io->client,
+    TNL_LOG(DEBUG, "intercepted address[%s] client[%s] service[%s]", io->tnlr_io->intercepted, io->tnlr_io->client,
```

Comment:
I'm on the fence about this, mainly because it now logs at a different level than when a tcp client is intercepted.